bcipy/display/paradigm/matrix/display.py

```diff
@@ -162,7 +163,7 @@ def build_grid(self) -> None:
             text_stim = visual.TextStim(
                 win=self.window,
                 text=sym,
-                opacity=self.start_opacity,
+                opacity=opacity if opacity else self.start_opacity,
```

Comment:
Python treats 0 as False, so this could lead to unexpected results if you try to set opacity to 0. Instead you might want to test if `opacity is None`.